.github/workflows/build-triton-wheel.yml

```diff
@@ -114,24 +119,32 @@ jobs:
           esac
 
           RELEASE=""
+          WITH_CLANG_LDD=""
           if [[ "${IS_RELEASE_TAG}" == true ]]; then
             RELEASE="--release"
           fi
 
           docker exec -t "${container_name}" yum install -y zlib-devel zip
-          docker exec -t "${container_name}" "${PYTHON_EXECUTABLE}"  -m pip install -U setuptools==67.4.0 pybind11==2.13.1
-          if [[ "${{ matrix.device }}" == "cuda" ]]; then
+          docker exec -t "${container_name}" "${PYTHON_EXECUTABLE}"  -m pip install -U setuptools==67.4.0 pybind11==2.13.1 auditwheel
+          if [[ "${{ matrix.device }}" == "cuda" &&  "${PLATFORM}" == "manylinux_2_28_x86_64" ]]; then
             # With this install, it gets clang 16.0.6.
             docker exec -t "${container_name}" dnf install clang lld -y
+            WITH_CLANG_LDD="--with-clang-ldd"
+          fi
+          docker exec -t "${container_name}" bash -c "${PYTHON_EXECUTABLE} /pytorch/.github/scripts/build_triton_wheel.py --device=$BUILD_DEVICE $RELEASE $WITH_CLANG_LDD"
+          if [[ "${{ matrix.device }}" == "cuda" ]]; then
+            docker exec -t "${container_name}"  bash -c "auditwheel repair --plat ${PLATFORM} //artifacts/*.whl"
```

Comment:
Is the output of auditwheel stored in `/artifacts/wheelhouse` dir by default?

Reply:
Yes